src/juxt/site/alpha/init.clj

```diff
@@ -239,6 +236,75 @@
 (def base-uri-parser
   (p/complete
    (p/into {}
-    (p/sequence-group
-     (p/pattern-parser #"(?<scheme>https?)://" {:group {:juxt.reap.alpha.rfc7230/scheme "scheme"}})
-     host-parser))))
+           (p/sequence-group
+            (p/pattern-parser #"(?<scheme>https?)://" {:group {:juxt.reap.alpha.rfc7230/scheme "scheme"}})
+            host-parser))))
+
+(defn do-action [crux-node subject action & args]
```

Comment:
Can be removed. Action system is not used in RLWS

Reply:
I guess that means the `src/pass/alpha/authorization.clj` file can also be removed as it only contains the called `do-action` function.